src/dtproject/types.py

```diff
@@ -126,9 +126,11 @@ class ContainerConfiguration(dict):
 class LayerContainers(DictLayer[ContainerConfiguration]):
     ITEM_CLASS = ContainerConfiguration
 
-
+@dataclasses.dataclass
 class DevContainerConfiguration(dict):
-    pass
+    docker_compose_file: str
+    service: str
```

Comment:
I have added the options from the devcontainer.json that are not present in docker-compose.yml, there are a couple that are present in both that I think is worth leaving in here (such as `forwardPorts`)